### ISSUE TITLE
NPS Survey: design enhancement

### ISSUE THREAD

**Issue by mdrovdahl** [Automattic/wp-calypso#34631]
The contrast on NPS survey popup is weak and probably doesn't meet accessibility standards.

#### What I expected
Better contrast.

#### What happened instead
Blurred vision =)

#### Browser / OS version
Chrome on ChromeOS Version 76.0.3809.38 (Official Build) beta (64-bit)

#### Screenshot / Video
![Screenshot 2019-07-12 at 10 33 32](https://user-images.githubusercontent.com/474467/61147844-461d0c00-a492-11e9-9649-2f289ad5b4a9.png)

#### Context / Source

cc: @drw158 who seems to have recent, related commits and @michiecat for good measure =)

**Comment by kriskarkoski** [comment]
Closing for now as NPS surveys were disabled earlier this month (p5uIfZ-9o5-p2) and we'll presumably revisit the design if/when we turn them back on.
